src/stories/RoundedIcon.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import RoundedIcon from 'components/RoundedIcon'
+import { Puzzle } from 'components/icons'
+
+export default {
+  title: 'Components/RoundedIcon',
+  parameters: {
+    layout: 'padded',
+  },
+}
+
+const Template = args => <RoundedIcon {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  icon: Puzzle,
+}
+
+export const DifferentColor = Template.bind({})
```

Comment:
And here it is DifferentColor

Reply:
You're right, I renamed it `OtherColor ` 🙏